### ISSUE TITLE
hover link on existing header is present in manager configuration

### ISSUE THREAD

**Issue by singharpita** [open-learning-exchange/planet#1582]
![manager_config](https://user-images.githubusercontent.com/27276132/42327919-b810642c-808c-11e8-82fa-3fe48e1605bc.png)
 also there is layout issue while hovering over text.

**Comment by lmmrssa** [comment]
@singharpita This is the stepper form where you are even able to click on tab to switch between. 
But yes it will be nice to have some UI fixes.
@paulbert If we do not want user to have tab being clicked then we can disable it from stepper. Do we want that?

**Comment by paulbert** [comment]
Let's wait on working on this issue until we start with #1587 so we know more about the next version of this form

**Comment by paulbert** [comment]
Outdated
